website/docs/cloud-docs/workspaces/health.mdx

```diff
@@ -60,6 +63,16 @@ Terraform Cloud pauses health assessments if the latest run ended in an errored
 
 Health assessments may take longer to complete when you enable health assessments in many workspaces at once or your workspace contains a complex configuration with many resources.
 
+<!-- BEGIN: TFC:only name:health-assessments -->
+### On Demand Assessments
+
+-> **Note:** On demand assessments are only available in the Terraform Cloud web UI. You can only trigger an on demand assessment if the workspace satisfies all assessment prerequisites.
```

Comment:
```suggestion
-> **Note:** On demand assessments are only available in the Terraform Cloud web UI.
```

opting to remove the extra line since we say essentially exactly that in the next paragraph!